docs/install.rst

```diff
@@ -304,53 +72,24 @@ To build the documentation from your virtual environment, first make sure that y
     pip install "Sphinx>=1.5.0"
     pip install sphinx-rtd-theme
     pip install sphinxcontrib-programoutput
-    
-To generate the documentation and push it to your personal GitHub pages, first create a branch names ``gh-pages``, if you do not already have one. Follow the `GitHub branch <https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/>`_ instructions to do this.
 
-To build and publish the documentation, run the following commands from the
-top-level of your PyCBC source tree, replacing ``github-username`` with your
-GitHub user name:
+You can then build the documentation locally as
 
 .. code-block:: bash
 
-    git clone git@github.com:github-username/pycbc.git _gh-pages
-    cd _gh-pages
-    git checkout gh-pages
-    git rm -rf *
-    git commit -a -m "flush documentation"
-    cd ..
-    python setup.py build_gh_pages
-    cd _gh-pages
-    git add --all
-    git commit -a -m "documentation update"
-    git push origin gh-pages
-
-The documentation will then be visible at http://github-username.github.io/pycbc/latest/html where ``github-username`` should be replaced with your GitHub username.
-
-.. note::
-
-    Be careful with the ``git rm -rf *`` command as if you run it in the wrong
-    directory you can delete the contents of your git repository. If you do
-    this by accident, you can use ``git reset`` to undo the commit.
-
-For more details on building and maintaining the documentation under GitHub project pages, see
-
-.. toctree::
-    :maxdepth: 1
-
-    build_gh_pages
+    python setup.py build_docs
```

Comment:
Where will the documentation be?

Reply:
I've added a sentence to say where they will show up.